scss/civicrm/administer/civi-case/_case-types.scss

```diff
@@ -7,11 +7,41 @@
       padding: 20px;
     }
 
+    input[type=checkbox] {
```

Comment:
i'm not sure what happened to the previous conversation, but I still don't know why we have CiviCase styles in shoreditch